gradle-6/dependency-constraints/src/main/java/com/baeldung/gradle/RxHelloWorld.java

```diff
@@ -0,0 +1,20 @@
+package com.baeldung.gradle;
+
+import com.google.common.collect.ImmutableList;
+import io.reactivex.Observable;
+
+import java.util.List;
+
+/** Demonstrates a library type that returns an RxJava type. */
```

Comment:
Let's use traditional javadoc here. Here is some information: https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#format

Reply:
I think I've addressed this feedback, but I struggled to find a way to automatically check this. The Oracle documentation you've linked references a DocCheck doclet for verifying formatting, but I cannot find this jar.

The Baeldung style guide does not specify that JavaDoc should be formatted.

```
<setting id="org.eclipse.jdt.core.formatter.comment.format_javadoc_comments" value="false"/>
```

 I changed this to `true` and formatted the project. Do the changes address your concern?